### PR TITLE
Updates benchmarked XCM instructions for Moonbase

### PR DESCRIPTION
**Pull request by albertov19**
### Description

Modifies the XCM Fees page to add the newly benchmarked XCM instructions.

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs-cn/pull/180

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/xcm/fees.md [hunk 1]
```diff
@@ -180,32 +183,59 @@ XCM-KSM-Cost = XCM-Planck-KSM-Cost / KSMDecimalConversion
...
+    
+=== "Moonriver"
+    | XCM Instruction Cost |
+    |:------------------------:|
+    | [{{ networks.moonriver.xcm.instructions.weight_units.display }}](https://github.com/PureStake/moonbeam/blob/f19ba9de013a1c789425d3b71e8a92d54f2191af/runtime/moonriver/src/xcm_config.rs#L208){target=_blank} |
+    
+=== "Moonbase Alpha"
+    | Benchmarked Instructions | Non-Benchmarked Instructions |
+    |:------------------------:|:-------------------------:|
+    | [Generic XCM Instructions](https://github.com/PureStake/moonbeam/blob/master/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_generic.rs#L203){target=_blank} | [Fungible XCM Instructions](https://github.com/PureStake/moonbeam/blob/master/pallets/moonbeam-xcm-benchmarks/src/weights/moonbeam_xcm_benchmarks_fungible.rs#L26){target=_blank} |
```
Comment (eshaben) [inline builders/xcm/fees.md:238]: are all of the fungible assets going to be benchmarked at the same time?

Comment (albertov19) [reply builders/xcm/fees.md]: This is not fungible assets but fungible instructions. And I would expect yes

Comment (albertov19) [reply builders/xcm/fees.md]: Fungible instructions means instructions that handle assents

Comment (eshaben) [reply builders/xcm/fees.md]: sorry meant to say instructions haha but ok

Comment (eshaben) [reply builders/xcm/fees.md]: otherwise I would say we should reformat this table but I guess its ok as it is

**Review by eshaben — APPROVED** [review]
LGTM

**Pull request merged**